### PR TITLE
Initial PR 

### PR DESCRIPTION
**Pull request by userin4o**
This PR includes the main layout and functionality for the frontend of the refbox. It implements fetching and displaying data from the Rest-Api.

## It includes:
 - ### The current layout of the refbox
 - ### Functionality
   * adding team name
   * fetching from Refbox-Rest-Api and displaying robots, machines, score, orders, team name, and game state information
   * polling from the Refbox-Rest-Api Endpoints of machines, orders, score and gamestate information to display the latest data
* ### Ordersinfo displayed:
     - productssvg corresponding to orderspecification
     - delivery -period
     - id, checkbox for delivery
* ### Robotsinfo displayed:
     -  host
     - state
     - numbers of maintenances
* ### Machineinfo displayed:
     - state
     - type 
     - zone
     - RS : bases used - bases added, rs ring colors and bases requered
* ### Score:
     - points and reasons for the awarded points

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
 # rcll-refbox-frontend
 A web frontend for the RCLL RefBox
+
+## Project setup
+```
+cd frontend/
```
Comment (snoato) [inline README.md]: the directory is called frontend-client

File: frontend-client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <!--Font Awesome Import-->
+    <script src="https://kit.fontawesome.com/b9f3787362.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab&display=swap" rel="stylesheet">
```
Comment (snoato) [inline frontend-client/public/index.html]: I'm not sure if for this project an external online dependency (Google Fonts and Font Awesome) is ideal. A local version of the font might be the better way to go in this situation, if there might be the chance that the refbox is not connected to the internet (or has a slow/unstable connection)

File: frontend-client/src/components/BodyMachinesTeamCyanGenerator.vue [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  },
+
+  methods: {
+    ...mapActions({
+      fetchMachinesCyan: 'fetchMachinesCyan',
+      fetchRingSpec: 'fetchRingSpec',
+    }),
+    pollMachineInfo() {
+      // Polls Information every 1,5 seconds
+      setInterval(this.fetchMachinesCyan, 1500)
```
Comment (snoato) [inline frontend-client/src/components/BodyMachinesTeamCyanGenerator.vue]: maybe you can add a variable for the polling wait time, I think that might be a bit more clean.

**Review by snoato — APPROVED** [review]
Very good job so far! There's only two things (besides the minor comments I added a few of the files) that I'd prefer to be different, both regarding the CSS handling:
- Personally I think it's nicer to have the Stylesheets in separated files, this could also save some redundancies. Of course it makes sense for the single file components, but that leads me to point two
- I dislike local changes to pre-defined semantical HTML Tags, e.g. changing the font-size locally for an h6 tag, that should always be done globally in my opinion.

**Comment by userin4o** [comment]
Dani, thanks for your feedback!
Based on your comments I made some changes!

As for the css styling, i would personally prefer to have the css styling scoped in it's single file component. Using Bootstrap the written css in the style scope is close to minimum. 
As for the second point, you have right. I will avoid changing any html tags.

**Pull request merged**